docs/trestle_author_jinja.md

```diff
@@ -111,8 +111,16 @@ Trestle provides to custom jinja tags for use specifically with markdown: `mdsec
 
 `mdsection_include` is similar to the native `md_clean_include` except that.:
 
-1. `mdsection_include` requires an second positional argument which is the title of a heading, from a markdown file, which you want the cotent from.
+1. `mdsection_include` requires an second positional argument which is the title of a heading, from a markdown file, which you want the content from.
+
    1. E.g:  `{% mdsection_include 'test_markdown.md' '# Header we want' %}`
+
 1. `mdsection_include` can be used with an optional keyword argument `heading_level` argument similar to `md_clean_include`
+
    1. `{% mdsection_include 'test_markdown.md' '# Header we want' %}`
    1. The heading level argument adjusts to (based on the number of hashes) the most significant heading in the chosen section, if headings exist.
+
+`md_datestamp` inserts a date stamp into the output. By default the date is in the format '%m/%d/%Y', e.g. '12/28/2001' and is followed by a double newline to prevent subsequent headings failing to parse correctly. E.g: `{% md_datestamp %}` results in a date in the format '12/28/2021' being inserted. `md_datestamp` can be used with the following optional keyword arguments:
+
+1. `format` where a python [datetime strftime format string](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes) is provided to format the output. E.g. `{% md_datestamp format='%B %d, %Y' %}` results in `December 28, 2021` being inserted.
+1. `newline` is a boolean to control the addition of a double newline after the inserted date string. For example `{% md_datestamp newline=false %}` inserts a date in the default format, without additional newlines.
```

Comment:
Isn't %Y-%m-%d a more common default?  I realize the user can change it.  Trestle currently mainly uses that if you look for strptime and strftime.